src/seedu/addressbook/AddressBook.java

```diff
@@ -1146,12 +1152,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefixSign(String fullString, String prefix) {
+        return fullString.replaceFirst(prefix, "");
```

Comment:
Shouldn't there be a validation to check if fullString.indexOf(prefix) == 0 before you actually do the replace?? If not the first occurrence of the prefix is what is getting replaced, right?